.github/workflows/ci.yml

```diff
@@ -69,22 +69,58 @@ jobs:
           DATADOG_APP_KEY: ${{ secrets.datadog_app_key }}
 
   standalone-binary-test:
-    name: Test standalone binary
+    name: Test standalone binary in ubuntu
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - name: Install node
         uses: actions/setup-node@v1
         with:
-          node-version: '14.15.3'
-      - uses: actions/setup-python@v2
+          node-version: '14.18.3'
+      - run: yarn install --immutable
+      - run: yarn build
+      - name: Bundle standalone binary
+        run: yarn dist-standalone -t node14-linux-x64 -o datadog-ci-linux
+      - name: Remove dist and src folder to check that binary can stand alone
+        run: |
+          rm -rf dist
+          rm -rf src
```

Comment:
Should it also remove `node_modules/` to ensure modules are properly embedded in the binary too?

Reply:
yeah, good idea

Reply:
oops, no, we can't do that. We still use `jest` for running the tests

Reply:
we could get rid of everything and then install `jest`, but we also need `ts-jest`, and maybe more. I'd keep it as is and think about it in the future, WDYT?

Reply:
Yes, let's keep it as is then! :+1: